helpdesk_mgmt/views/helpdesk_ticket_templates.xml

```diff
@@ -11,10 +11,8 @@
                 t-if="page_name == 'ticket' or ticket"
                 t-attf-class="breadcrumb-item #{'active ' if not ticket else ''}"
             >
-                <a
-                    t-if="ticket"
-                    t-attf-href="/my/tickets?{{ keep_query() }}"
-                >Tickets</a>
+                <a t-if="ticket" t-attf-href="/my/tickets?{{ keep_query() }}">Tickets
```

Comment:
This diff continues. Please review your pre-commit config

Reply:
I haven't change any pre-commit config,

Now I push code as it was and its pre-commit failed